### DISCUSSION TITLE
How to effectively initiate multiple client links？

### DISCUSSION THREAD

**Discussion by thisIBaby** [netty/netty#11775] [Q&A]
How to effectively initiate multiple client links？

**Comment by NiteshKant** [comment]
Not sure what do you mean by "links" here, can you elaborate?

**Reply by thisIBaby** [reply]
Multiple netty client  ,   connect   to   Multiple servers. How does my code change work，thanks

**Comment by NiteshKant** [comment]
Looks duplicate of this: https://github.com/netty/netty/issues/11769#issuecomment-945437449
